LearningMissionSimulation/PlaygroundVahe.cs

```diff
@@ -86,32 +92,72 @@ public void ActivateAccounts()
             }
         }
 
-        public void AssignModulesToInstructors()
+        public void CreateSubjects(int subjectCount)
         {
-            throw new NotImplementedException();
+            for (int i = 0; i < subjectCount; i++)
+            { 
+                Subject subject = ObjectGenerator.GenerateSubject();
+                subjectList.Add(subject);
+                subjectIdList.Add(subject.Id);
+            }
         }
 
-        public void AssignModulesToStudents()
+        public void CreateModules(int moduleCount)
         {
-            throw new NotImplementedException();
+            for (int i = 0;  i < moduleCount; i++)
+            {
+                Guid subjectId = subjectIdList[AttributeGenerator.random.Next(0, subjectIdList.Count)];
+                Module module = ObjectGenerator.GenerateModule(subjectId);
+                moduleDictionary.Add(module.Id, module);
+                moduleIdList.Add(module.Id);
+            }
         }
 
-        public void CreateClassrooms(int classroomCount)
+        public void AssignModulesToInstructors()
         {
-            throw new NotImplementedException();
+            foreach (var instructor in instructorList)
+            {
+                instructor.ModuleList = GetModuleList();
+            }
         }
 
-        public void CreateModules(int moduleCount)
+        public void AssignModulesToStudents()
         {
-            throw new NotImplementedException();
+            foreach (var student in studentList)
+            {
+                student.CompletedModuleList = GetModuleList();
+            }
         }
 
-        public void RegisterStudentsForClasses()
+        List<Module> GetModuleList()
+        {
+            List<Module> moduleList = new List<Module>();
+            int moduleCount = moduleIdList.Count;
+            int maxModuleCountLimit = 7;
```

Comment:
Move:
            int maxModuleCountLimit = 7;
            int minModuleCountLimit = 1;
to SimulationConstants